dev-docs/Cloud-Env-Testing.md

```diff
@@ -121,3 +121,36 @@ Before running the script, ensure that:
 
 **Note**: The script will ask for confirmation before deleting each environment, unless you set the `interactive` flag
 to `false`.
+
+### Delete Environment Workflow
+
+The new **Delete Environment** GitHub workflow automates the process of cleaning up environments. When activated, it automatically performs the cleanup of environments, ensuring that all associated resources are correctly removed.
+
+#### How to Run the Flow
+
+Follow these steps to run the workflow:
+
+1. Go to [`Actions > Destroy Environment`](https://github.com/elastic/cloudbeat/actions/workflows/destroy-environment.yml)
+
+   ![Destroy Environment](https://github.com/gurevichdmitry/cloudbeat/assets/99176494/505d6553-7780-4450-83e9-3617f64442ad)
+
+2. Click the `Run workflow` button.
+
+   ![Run Workflow](https://github.com/gurevichdmitry/cloudbeat/assets/99176494/8965311c-eeac-492f-a564-a57c46854a3a)
+
+3. Complete the required input fields:
+
+    - `ec-api-key` (required): Use the [Production Elastic Cloud](https://cloud.elastic.co/home) API KEY.
+    - `prefix` (required): The prefix used to identify the environments to be deleted.
+
+   ![Enter Inputs](https://github.com/gurevichdmitry/cloudbeat/assets/99176494/dde955f1-21fe-4cb1-a7bc-674d8969b001)
```

Comment:
can we resize the image size? https://gist.github.com/uupaa/f77d2bcf4dc7a294d109

Reply:
Updated

Reply:
new images uploaded